### ISSUE TITLE
Exhibit item iframe doesn't have title attribute

### ISSUE THREAD

**Issue by thanyabegum** [pulibrary/dpul#1219]
The ```<iframe>``` of an exhibit item doesn't have a title attribute describing its contents.

<img width="531" alt="Screen Shot 2021-11-05 at 3 04 36 PM" src="https://user-images.githubusercontent.com/66143640/140564998-b19c7f67-0fff-4955-b926-7e11e9a743ad.png">

[Learn more about this failing Lighthouse test.](https://web.dev/frame-title/?utm_source=lighthouse&utm_medium=devtools)

**Comment by kelea99** [comment]
This seems important to fix, especially if screen readers rely on it for contents of the frame. @sdellis I defer to you, and others. What are your thoughts?
